tests/tensor/random/test_op.py

```diff
@@ -22,20 +22,24 @@ def set_aesara_flags():
         yield
 
 
-def test_default_shape_from_params():
+def test_default_supp_shape_from_params():
```

Comment:
Reading through all the case docstrings that  explain whats going on would be nice, even if its "obvious" to you folks itlll make it easier to understand the functionality for people ramping in